### PR TITLE
Feature/why focused

### PR DESCRIPTION
**Pull request by NSeydoux**
Added new flavors of "Why you'll love Solid". We can use this PR to discuss the copy before merging.

- As a user
  - Move freely between services: Switch to the best tools and <b>take your data with you</b>
  - Reuse data across apps: Stop re-entering the same data again and again by <b>recycling data from app to app</b>.
  - Connect with anyone: Connect to your friends <b>no matter what app they are using</b>. 
  - Select what you share precisely: Give access to <b>the bare minimum of data needed</b> for the service to work.
- As a developer
  - Don't start from scratch: Reuse data <b>already available</b> from your user 
  - Feel free to innovate faster: Experience <b>permissionless innovation</b> by going directly to users for their data
  - Built-in sharing, greater value: Empower your user to <b>make your application even more valuable to them</b>
  - Contribute to the whole stack: The Solid standards are <b>open</b>, as well as some of their implementations.
- As an enterprise:
  - Focus on the service: Let the customer choose where to host their data <b>depending on their preferences</b>.
  - Don't compete for the data: When data is reusable, you can <b>compete against installed actors</b> by offering a better service 
  - Get portability without hassle: If you host your company data on Solid, you can easily <b>move between providers</b>. 
  - Build trust with your customers: Show to your customer the data that concerns them, and help them protect it.

why-love

### CODE REVIEW

**Comment by NSeydoux** [comment]
Don't worry about being blunt, you are completely right :). It's true that I didn't think about accessibility in that case, I'll try to figure out a different way to convey the message to the different audiences.

As for the developers items, do you have in mind elements that could be seen as selling points for Solid particularly relevant to developers ?

**Comment by Mitzi-Laszlo** [comment]
Perhaps we could put the 'why you'll love Solid' for developers in the 'for developers' tab. 

Ultimately everyone is a user so it makes sense to have that at the forefront on the home page.

**Comment by Vinnl** [comment]
> As for the developers items, do you have in mind elements that could be seen as selling points for Solid particularly relevant to developers ?

Not particularly, though I think many of the current ones are mostly aimed at businesses? Things I like:

- Not needing to maintain a separate back-end, and the associated logging, monitoring, etc.
- Related: not having to wait on a back-end team to implement features I need as a front-end developer.
- Fewer worries about scalability.
- My boss wants me to build something on Solid :P

**Comment by NSeydoux** [comment]
I did not update the copy yet, but I scattered the "Why you'll love" cards across the website so that the first one you run into is still the one for users (as suggested by Mitzi), and so that the other ones are featured respectively on the for-developers and the for-enterprises pages (I added the latter, which contains the text of the /implement page).

why-love

**Comment by NSeydoux** [comment]
Using @Vinnl input, I changed the copy of the developers and enterprises "why you'll love": 
- Developers:
  - Focus on the essential: With **application decoupled form data**, no need to maintain a separate back-end and what comes with it (logging, monitoring...) 
  - Go directly to the data: Don't wait on an API provider to let you **access the data you need**: go for it directly.
  - Be part of a community: Reuse other apps' data, and let other reuse yours. 
  - Contribute to the whole stack:  The Solid standards are open, as well as some of their implementations. 

How I understood it (and correct me if I'm wrong), the question of scalability was overlapping with the separation from the back-end. Moreover, since we're still unclear on how scalable Solid will be, I wasn't sure how to present it ;)

- Enterprises:
  - Don't start from scratch: Reuse data already available from your user, and compete with other apps for a better service rather than for data. 
  - Built-in sharing, greater value: Empower your user to make your application even more valuable to them by reusing the service you provide. 
  - Build trust with your customers: Let your customers help you managing the data relevant to them, and help them protecting it. 
  - Get portability without hassle: If you host your company data on Solid, you can easily move between providers.

**Comment by NSeydoux** [comment]
Related to issue #181

File: _includes/why2love_developer.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <img
+          src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/solid/crop-alt.svg"
+          alt=""
+          class="icon"
+        />
+        <span class="card-title">Focus on the essential</span>
+      </div>
+      <div class="card-body">
+        <p>
+        With <b>application decoupled form data</b>, no need to maintain a separate back-end and what comes with it (logging, monitoring...)
```
Comment (Vinnl) [inline _includes/why2love_developer.html]:
```suggestion
        With <b>applications decoupled from their data</b>, there is no more need to maintain a separate back-end, nor infrastructure for logging, monitoring, etc.
```

**Comment by Mitzi-Laszlo** [comment]
Perhaps an idea to include the emotion in the short copy and then start the long description with a verb that hits the message. For example: 

Developers:
* Permissionless innovation: Build apps that work straight away because users can give access to the complete datasets directly
* Focus on your idea: Get going straight away with building your app without needing to set up a backend 
* Do the right thing: Contribute towards building solutions that allow users to self-determine.

Enterprises:
* Do the right thing: Contribute towards building solutions that allow users to self-determine. 
* Get a leg up: Tap into recycled data to increase efficiency and accuracy of insights. 
* Seamless portability: Comply to GDPR portability regulation conveniently.

**Comment by Mitzi-Laszlo** [comment]
Also, let's put the 'why you'll love Solid' for developers on https://solidproject.org/for-developers 

and  'why you'll love Solid' for enterprises on https://solidproject.org/implement

**Comment by NSeydoux** [comment]
At this point, I'm not sure that `/implement` is still relevant. I moved all of its content into `/for-enterprises`, which caters to a more specific audience, and makes it clear at a first glance that the visitors of the website belong to one of three categories: users (i.e. end users), to whom the landing page is targeted, and who can get more info in the `Use Solid` tab, and developers and enterprises (actually, organisations generally, but I don't know if it's relevant to make this distinction right away), whom each have a dedicated page clearly accessible from the start, with the associated "Why you'll love"

**Comment by NSeydoux** [comment]
Following @Mitzi-Laszlo recommandations to start with verbs to leverage the message from the card header, "Why you'll love" for developers now reads:
- Focus on your idea: Build applications decoupled from data, and avoid maintaining a separate back-end and what comes with it (logging, monitoring...) 
- Go directly to the data: Access the data you need straight from the user, without depending on any API provider-specific feature.
- Be part of a community: Reuse other apps' data, and let other reuse yours. 
- Contribute to the whole stack: Be empowered to read and improve the specification: the Solid standards are open, as well as some of their implementations. 

and Why you'll love for enterprises: 
- Get a leg up: Tap into data already available from your user, and compete with other apps for a better service rather than for data. 
- Built-in sharing, greater value: Empower your user to make your application even more valuable to them by reusing the service you provide.
- Build trust with your customers: Let your customers help you managing the data relevant to them, and help them protecting it. 
- Get portability without hassle: Comply to GDPR portability regulation conveniently, or chose the best storage for your company data.

You'll notice that I didn't keep the "Do the right thing" item, because even though I think that using Solid IS doing the right thing, and I also think most people are keen on doing the right thing, I always feel like this is a weak argument, that sounds like what we propose is not as good as the rest, but at least it is ethical, which I feel is not right in this case: using Solid does help building better stuff, doing the right thing is the cherry on top :). However, I'm completely open to putting this argument in the mix if you think it is a point to get across of higher importance than the ones listed.

**Comment by Vinnl** [comment]
A bit of input on "for developers" in our first interview today:

- no hassle setting up a sign-up flow
- USP for users
- be an early adopter
- no need to give Facebook a competitive advantage by giving it access to your user's data

**Pull request merged**